src/iexamine.cpp

```diff
@@ -1559,7 +1558,7 @@ std::list<item> iexamine::get_harvest_items( const itype &type, const int plant_
         return result;
     }
     const islot_seed &seed_data = *type.seed;
-    const itype_id &seed_type = type.id;
+    const itype_id &seed_type = item( &type ).typeId();
```

Comment:
Can be resolved nicely via moving `get_harvest_items(...)` into `item` (where it probably belongs).